addons/addon-internal-auth-dep-check/packages/internal-auth-dep-check/lib/steps/internal-auth-dep-check-service.js

```diff
@@ -42,6 +42,20 @@ class InternalAuthDepCheckService extends Service {
   async execute() {
     // Keep track of resources blocking upgrade
     const blockers = {};
+    const usersTable = this.settings.get(settingKeys.usersTableName);
+
+    // check not fresh install
+    try {
+      await this.dbService.dynamoDb.describeTable({ TableName: usersTable }).promise();
+    } catch (e) {
+      if (e.code === 'ResourceNotFoundException') {
+        this.log.info('This is first time deployment, no resources exist.');
+        return [];
```

Comment:
Why return empty list? (why not simply `return;` ?)

Reply:
following the convention in the other pre-dep check! I can simply `return;` though!